tests/functional/add-mdss/container/vagrant_variables.yml

```diff
@@ -51,7 +51,7 @@ disks: "[ '/dev/sdb', '/dev/sdc' ]"
 # For more boxes have a look at:
 #   - https://atlas.hashicorp.com/boxes/search?utf8=✓&sort=&provider=virtualbox&q=
 #   - https://download.gluster.org/pub/gluster/purpleidea/vagrant/
-vagrant_box: centos/7
+vagrant_box: centos/stream8
```

Comment:
Is the Quincy container based on stream8?
I'm asking because I'm wondering if we would like to bump the host to stream-9. It's fine having stream8, in OSP we started testing pacific on stream8, but today we bumped our CI to stream9 only [1], and I'm expecting to test Quincy in that scenario as well.
We have also situations where the underlying OS (in case of upgrades) is left behind (stream8), so your choice here makes a lot of sense to me.

[1] https://review.opendev.org/c/openstack/tripleo-heat-templates/+/778915

Reply:
> Is the Quincy container based on stream8? 

Yes it is.

> I'm asking because I'm wondering if we would like to bump the host to stream-9. It's fine having stream8, in OSP we started testing pacific on stream8, but today we bumped our CI to stream9 only [1], and I'm expecting to test Quincy in that scenario as well. We have also situations where the underlying OS (in case of upgrades) is left behind (stream8), so your choice here makes a lot of sense to me.
> 
> [1] https://review.opendev.org/c/openstack/tripleo-heat-templates/+/778915

This change affects the upstream CI only, I think it will make sense at some point to test against stream9 hosts anyway.